Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.yml

```diff
@@ -2255,10 +2279,70 @@ script:
       task with the specified ID. Used for deploying.
     execution: false
     name: ciscofp-get-task-status
+    arguments:
+    - name: task_id
+      required: true
+      description: The ID of the task for which to check the status.
     outputs:
     - contextPath: CiscoFP.TaskStatus.Status
       description: task status
       type: String
+    description: Retrieves information about a previously submitted pending job or
+      task with the specified ID. Used for deploying.
+name: ciscofp-get-url-groups-object
+    arguments:
+    - name: id
+      required: true
```

Comment:
seems like you broke something in the yml: "name" field is not indented correctly and you have some missing details in the "id" argument (in both cases, it was ok in my commits)
please fix and feel free to reach me on Slack if you need help.